src/typesetting/PSandAI.md

```diff
@@ -0,0 +1,30 @@
+# Masking using AI and PS
+# Preface
```

Comment:
You should never have two headings of the same level next to each other. This should probably be a h2 or be combined into one heading.

Reply:
I have combined it since it makes more sense to me that way